lib/mail_room/configuration.rb

```
@@ -8,17 +8,20 @@ class Configuration
     def initialize(options={})
       self.mailboxes = []
 
-      if options.has_key?(:config_path)
-        config_file = YAML.load_file(options[:config_path])
+      return unless options.has_key?(:config_path)
+      return unless File.exist?(options[:config_path])
```

Also, silently failing if the config path given does not exist.